src/include/rbd/librbd.hpp

```diff
@@ -101,6 +101,12 @@ namespace librbd {
     time_t deferment_end_time;
   } trash_image_info_t;
 
+  typedef struct {
+    std::string pool_name;
+    std::string image_name;
+    std::string image_id;
+  } children_info_t;
```

Comment:
Perhaps add a ```bool trash``` member so that users don't need to re-execute a ```trash_get``` (since it was already performed within the API).